tests/CMakeLists.txt

```diff
@@ -92,8 +92,7 @@ endforeach()
 ###################
 # DLT daemon tests
 ###################
-set(TARGET_LIST gtest_dlt_daemon_gateway
```

Comment:
what is the purpose of removal here? what makes this test suite different with the other?